src/client/containers/FavoritePage/FavoritePage.Container.js

```diff
@@ -0,0 +1,78 @@
+
+import './FavoritePage.styles.css';
+
+   
+import React, { useState, useEffect, useCallback } from "react";
+import FavouriteList from '../../components/FavouriteList/FavouriteList.component';
+
+
+
+export const addProductToFavorites = (user_id,product_id) => {
+  console.log("in add favorites container");
+  console.log(user_id,product_id)
+ 
+  const requestOptions = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        product_id:product_id,
+        user_id: user_id
+        
+       }),
+       
+    };
+    fetch("/api/favorites/add", requestOptions)
+      .then((response) => response.json())
+      .then((result) => {
+        console.log(result);
+      });
+  };
```

Comment:
I don't think this post request is actually working. I could not add any new products in the favourite list.  I tried the following. Maybe you can find a better solution.
`export const addProductToFavorites = async (userId, productId) => {

  const response = await fetch('/api/favorites/add', {
    method: 'POST',
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({
      product_id: productId,
      user_id: userId,
    }),
  });
  await response.json();
  if (response.status === 200) {
    alert('Item added to favorites'); // In stead of alert we will have to show the modal.
  } else {
    alert('Received error');
    console.error(
      'Received unexpected error from api/favorites/add',
      response.status,
    );
  }
};`

Reply:
Hi, the code previously written in branch is working. Can you show the error you are getting when you are running it

Reply:
@FlorentinaPetica I think, it might be not working because your user_id updated to uid! Please check the users table!.
And I am also not able to check its UI because of the same problem.

Reply:
@FlorentinaPetica does it work for you now? I had issues stemming from using Mysql 5.x.